ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js

```diff
@@ -23,17 +23,29 @@ const CustomElement = styled.div`
 const Visualization = styled.div`
   cursor: pointer;
   opacity: ${props => props.theme.minimumOpacity}
+  padding: ${props => props.theme.largeSpacing}
   :hover {
     opacity: 1
   }
 `
-const VisualizationChoice = styled.div`
-    white-space: nowrap;
-    padding: ${props => props.theme.largeSpacing}
-    cursor: move;
-    cursor: grab;
-    cursor: -moz-grab;
-    cursor: -webkit-grab;
+const Visualization = styled.div`
+  cursor: pointer;
+  opacity: ${props => props.theme.minimumOpacity};
+  padding: ${props => props.theme.largeSpacing};
+  :hover {
+    opacity: 1;
+  }
+  white-space: nowrap;
+  cursor: move;
+  cursor: grab;
+  cursor: -moz-grab;
+  cursor: -webkit-grab;
+`
+  white-space: nowrap;
```

Comment:
Looks like accepting my change orphaned this block.  Not really sure how suggestions work 🤔